src/utils/fetchAPI.js

```diff
@@ -0,0 +1,21 @@
+import getApiEndpoint from './apiEndpoint';
+
+const fetchAPI = async (path, params = {}) => {
+  const headers = params.headers ? params.headers : {};
+  const parameters = {
+    ...params,
+    headers: {
+      ...headers,
+      Authorization: 'Bearer admin',
```

Comment:
Our extra headers needs to be used both here and in our socket requests. I would define them in a third file, and import it from there.